content/kubermatic/master/tutorials_howtos/networking/ipam/_index.en.md

```diff
@@ -0,0 +1,157 @@
++++
+title = "Multi-Cluster IP Address Management (IPAM)"
+date = 2022-07-26T14:45:00+02:00
++++
+
+Feature responsible for automating the allocation of ranges of IP addresses per-user-cluster, based on a predefined configuration ([IPAMPool](#input-resource-ipampool)) per datacenter that defines the pool subnet and the allocation size. The user cluster allocated ranges are available in the [KKP Addon](#kkp-addon-template-integration) `TemplateData`, so it can be used by various applications / Addons running in the user cluster.
+
+{{< table_of_contents >}}
+
+## Motivation and Background
+Networking applications deployed in KKP user clusters need automated IP Address Management (IPAM) for IP ranges that they use, in a way that prevents address overlaps between multiple user clusters. An example for such an application is MetalLB load-balancer, for which a unique IP range from a larger CIDR range needs to be configured in each user cluster in the same datacenter. This is currently mostly done manually, by keeping evidence of allocated subnets in an external document and copy-pasting from it when deploying the MetalLB addon.
+
+The goal is to provide a simple solution that is automated and less prone to human errors.
+
+## Allocation types
+Each IPAM pool in a datacenter should define an allocation type: "range" or "prefix".
+
+### Range
+Results in a set of IPs based on an input size.
+
+E.g. the first allocation for a range of size **8** in a pool subnet `192.168.1.0/26` would be
+```txt
+192.168.1.0-192.168.1.7
+```
+
+*Note*: There is a minimum allowed pool subnet mask based on the IP version (IPv4 or IPv6). So, if you need a large range of IPs, it's recommended to use the "prefix" type.
```

Comment:
Please mention what is the limit exactly.

Reply:
true